test/testdata/rbs/signatures_defs.rb

```diff
@@ -181,6 +181,39 @@ def method20(&block)
   T.reveal_type(block) # error: Revealed type: `T.untyped`
 end
 
+# Some comment
+#: (String) -> void
+
+def method21(x)
```

Comment:
Can you also add a nested defs example?

Reply:
I added a nested def in this file however we also want to test for without `begin`, I think I'll create a file on its own and update our implementation to recurse more, see if it's fast enough.